ruby/generate-website-screenshot/index.rb

```diff
@@ -0,0 +1,41 @@
+require 'net/http'
+require 'addressable/uri'
+require 'json'
+require 'base64'
+require 'open-uri'
+
+def main(request, response)
+    website_url = nil
+    begin
+        payload = JSON.parse(request.payload)
+        website_url = payload["url"]
+    rescue Exception => e
+        return response.json({"message":"Payload is invalid."})
```

Comment:
the response should also include `status` as `false`.

Reply:
Oh yeah, I will correct that too.